app/assets/stylesheets/app.scss

```diff
@@ -0,0 +1,185 @@
+body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: 'ef_cuisinesregular';
+  src: asset_url('efcuisines-regular-webfont.eot');
+  src: asset_url('efcuisines-regular-webfont.eot?#iefix') format('embedded-opentype'),
+  asset_url('efcuisines-regular-webfont.woff') format('woff'),
+  asset_url('efcuisines-regular-webfont.ttf') format('truetype'),
+  asset_url('efcuisines-regular-webfont.svg#webfont') format('svg');
+}
+
+.banner {
+  background: url("banner_background.jpg") no-repeat center;
+  background-size: cover;
+  height: 350px;
+  display: flex;
+  flex-direction: column;
+}
+.header-wrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  height: 75%;
+}
+
+.add-restaurant {
```

Comment:
you can separate to different scss files according to components

Reply:
👍